pages/docs/tutorials/generate-code.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Generate code 
+description: In this tutorial, you'll learn how to generate code from your AsyncAPI document.
+weight: 130
+---
+
+## Introduction
+
+In this tutorial, you'll learn how to generate code from your AsyncAPI document using the AsyncAPI generator tool.
+
+## Background context
+The [AsyncAPI Generator](https://github.com/asyncapi/generator) is a tool that you can use to generate whatever you want based on the AsyncAPI document. You can generate docs and code. It can be used as a library in Nodejs application or through the [AsyncAPI CLI](https://github.com/asyncapi/cli).
+
+The generator tool supports a number of templates to generate code for a variety of different languages and protocols as the output. These templates help to specify what exactly must be generated, and in this tutorial, you use [Nodejs template](https://github.com/asyncapi/nodejs-template).
+
+## Installation guide
+Before proceeding to the next step, 
+
+<Remember>
+
+It is important to have [Node.js](https://nodejs.org/en/download/) installed which will enable the installation of the required dependencies using <b>npm</b>.
```

Comment:
So you can't use markup syntax inside a **Remember* `react component`. 

You need to update the link markup to have approved HTML `<a>` tags.